### PR TITLE
SUIT-18069 - getAppConfig function was added

### PR DESCRIPTION
**Pull request by Noorama**

### CODE REVIEW

File: lib/commands/getAppConfig.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const wsContentTypes = require('../api/wsContentTypes');
+const chainPromise = require('../utils/chainPromise');
+const {getAppConfigRequest, getAppConfigResponse, getAppConfigError} = require('../texts');
+
+async function getAppConfig({appContext, authContext, webSockets, logger}) {
+	logger.log(getAppConfigRequest(appContext._context.configId));
```
Comment (polomsky) [inline lib/commands/getAppConfig.js:6]: is this debug code?

Comment (Noorama) [reply lib/commands/getAppConfig.js:6]: No. This code is display corresponding progress message to console, while code is running.

File: lib/commands/getAppConfig.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		type: wsContentTypes.getAppConfig,
+		configId: appContext._context.configId,
+	}, getAppConfig.name);
+
+	const response = await webSockets.send(authedContent);
+
+	if (response.result === 'error') {
+		logger.error(getAppConfigError());
+
+		return undefined;
```
Comment (JeKachan) [inline lib/commands/getAppConfig.js:18]: According to type in index.d.ts, function should no return undefined value, but shoud throw an error.

File: lib/commands/getAppConfig.js (hunk 1 above)
Comment (JeKachan) [inline lib/commands/getAppConfig.js:6]: Not sure that we should do any logging for this function - this is just utility function, that do not affect test execution.

**Review by polomsky — APPROVED** [review]
I think everything should be in single commit, but just saying.

**Review by JeKachan — APPROVED** [review]

**Review by JeKachan — APPROVED** [review]

**Pull request merged**